about.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+
+    <title>PAN African Cloud Computing Summit 2022</title>
+    <link rel="stylesheet" href="about.css">
+</head>
+
+<body>
+    <header class="header-container">
+        <nav class="header-nav">
+            <div class="container-top-nav">
+                <div class="header-top-nav">
+                    <ul class="header-top-nav-desktop">
+                        <li><a href="./about.html"><i class="fab fa-facebook-f"></i></a></li>
+                        <li><a href="./about.html"><i class="fab fa-twitter"></i></a></li>
+                        <li><a href="./about.html">English</a></li>
+                        <li><a href="./about.html">My Page</a></li>
+                        <li><a href="./about.html">Logout</a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="container-nav">
+                <div class="logo-nav">
+                    <a href="./index.html">
+                        <img src="./Images/PAN_Logo.jpg" alt="logo of CC Global">
+                    </a>
+                </div>
+                <ul class="header-nav-desktop">
+                    <li><a href="./index.html">Home</a></li>
+                    <li><a href="./about.html">Program</a></li>
+                    <li><a href="./about.html">Join</a></li>
+                    <li><a href="./about.html">Sponsor</a></li>
+                    <li><a href="./about.html">News</a></li>
+                    <li><a href="./about.html">PANACCS Campaign</a></li>
+                </ul>
+            </div>
+        </nav>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if the navigation options you use here match sections of the `About` page i.e. once a user clicks on one option, he should be redirected to a specific section of the page(e.g.: See the past PANACCS Summits,...)